doc/nrf/protocols/matter/end_product/configuring_cd.rst

```diff
@@ -0,0 +1,83 @@
+.. _ug_matter_device_configuring_cd:
+
+Configuring Certification Declaration in firmware
+#################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Certification Declaration (CD) is :ref:`one of the documents <ug_matter_device_certification_results>` obtained as a result of a successful Matter certification.
+It is |matter_cd_definition|.
+It is used for this purpose in the :ref:`ug_matter_device_attestation` procedure during commissioning, where it is provided to a device as an element of :ref:`Device Attestation data <ug_matter_device_attestation_device_data>`.
+
+You must regenerate the CD for each new software version after you obtained the certification.
```

Comment:
You don't need to regenerate the CD but you need to update CD in your software, after obtaining it from CSA (after certification)

Reply:
@ArekBalysNordic wrote last week CD must be regenerated -- see his long answer to my first comment in this PR, with the following first line:

> Our customer must regenerate the CD for each new software version.